<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PharmacistFrame</class>
 <widget class="QFrame" name="PharmacistFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>657</height>
   </rect>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string notr="true">Pharmacist search</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lastLabel">
          <property name="toolTip">
           <string comment="PHARMACIST LAST SEARCH TOOLTIP">PHARMACIST LAST</string>
          </property>
          <property name="text">
           <string comment="PHARMACIST LAST SEARCH LABEL">PHARMACIST LAST</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lastField">
          <property name="toolTip">
           <string comment="PHARMACIST LAST SEARCH TOOLTIP">PHARMACIST LAST</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="firstLabel">
          <property name="toolTip">
           <string comment="PHARMACIST FIRST SEARCH LABEL">PHARMACIST FIRST</string>
          </property>
          <property name="text">
           <string comment="PHARMACIST FIRST SEARCH LABEL">PHARMACIST FIRST</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="firstField">
          <property name="toolTip">
           <string comment="PHARMACIST FIRST SEARCH TOOLTIP">PHARMACIST FIRST</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="activeField">
          <property name="text">
           <string notr="true">Active</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string notr="true">Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string notr="true">Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="newButton">
       <property name="text">
        <string notr="true">New Pharmacist</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="modifyButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">Modify Pharmacist</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="resultTable">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>pharmacists.id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="PHARMACIST LAST DISPLAY LABEL">PHARMACIST LAST</string>
      </property>
      <property name="toolTip">
       <string comment="PHARMACIST LAST DISPLAY TOOLTIP">PHARMACIST LAST</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="PHARMACIST FIRST DISPLAY LABEL">PHARMACIST FIRST</string>
      </property>
      <property name="toolTip">
       <string comment="PHARMACIST FIRST DISPLAY TOOLTIP">PHARMACIST FIRST</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="INITIALS DISPLAY LABEL">INITIALS</string>
      </property>
      <property name="toolTip">
       <string comment="INITIALS DISPLAY TOOLTIP">INITIALS</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="newAction">
   <property name="text">
    <string notr="true">New Pharmacist</string>
   </property>
   <property name="toolTip">
    <string notr="true">Enter a new pharmacist</string>
   </property>
  </action>
  <action name="modifyAction">
   <property name="text">
    <string notr="true">Modify Pharmacist</string>
   </property>
   <property name="toolTip">
    <string notr="true">Modify details of the pharmacist</string>
   </property>
  </action>
  <action name="toggleAction">
   <property name="text">
    <string notr="true">Toggle Active</string>
   </property>
   <property name="toolTip">
    <string notr="true">Toggle active/inactive</string>
   </property>
  </action>
  <action name="viewAction">
   <property name="text">
    <string notr="true">View Pharmacist</string>
   </property>
   <property name="toolTip">
    <string notr="true">View the pharmacist record</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lastField</tabstop>
  <tabstop>firstField</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>newButton</tabstop>
  <tabstop>modifyButton</tabstop>
  <tabstop>resultTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>lastField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>firstField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>firstField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newButton</sender>
   <signal>clicked()</signal>
   <receiver>newAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modifyButton</sender>
   <signal>clicked()</signal>
   <receiver>modifyAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>882</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
